packages/server/listenHandler.ts

```diff
@@ -10,8 +11,12 @@ const listenHandler = (listenSocket: us_listen_socket) => {
   if (listenSocket) {
     Logger.log(`\n🔥🔥🔥 Server ID: ${SERVER_ID}. Ready for Sockets: Port ${PORT} 🔥🔥🔥`)
     getGraphQLExecutor().subscribe()
-    // Cleaning on startup because shutdowns may be abrupt
-    serverHealthChecker.cleanUserPresence().catch(sendToSentry)
+    setTimeout(() => {
+      // if shutdowns are clean, this isn't necessary
+      // that's why we wait 3 minutes to let all the old servers shut down gracefully
+      serverHealthChecker.cleanUserPresence().catch(sendToSentry)
+      //
+    }, ms('3m'))
```

Comment:
Hey @mattkrick why not using the var [WEB_SERVER_RECONNECT_WINDOW](https://github.com/ParabolInc/parabol/blob/master/.env.example#L59) maybe + some seconds? That var has a default value of 60s if not declared, as stated in the doc. After that window, all web servers should already be stopped

Reply:
Great! I just wanted to be sure I wasn't missing something 😸